azure_monitor/src/azure_monitor/protocol.py

```diff
@@ -252,17 +365,17 @@ class RemoteDependency(BaseObject):
 
     def __init__(
         self,
-        ver=2,
-        name="",
-        id="",
-        result_code="",
-        duration="",
-        success=True,
-        data=None,
-        type=None,
-        target=None,
-        properties=None,
-        measurements=None,
+        ver: int = 2,
+        name: str = "",
+        id: str = "",
+        result_code: str = "",
+        duration: str = "",
+        success: bool = True,
+        data: Data = None,
+        type: str = None,
+        target: str = None,
+        properties: typing.Dict[str, any] = None,
+        measurements: any = None,
```

Comment:
I believe `measurements` is a dict.

https://github.com/microsoft/ApplicationInsights-Home/blob/master/EndpointSpecs/Schemas/Bond/RemoteDependencyData.bond#L51